_posts/2022-02-09-clustering-r.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Clustering with R"
+text: "An introduction to clustering methods with R.
+"
```

Comment:
Can probably remove these newlines in `text`?

Reply:
Can do!